app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdScreen.kt

```diff
@@ -303,7 +304,9 @@ class ScanSimpleIdScreen : BaseScreen<
   }
 
   override fun showInvalidQrCodeError() {
-    // Will handle in a later commit
+    val invalidQrCodeSnackbar = Snackbar.make(binding.root, R.string.scansimpleid_invalid_qr_code, Snackbar.LENGTH_SHORT)
+
+    invalidQrCodeSnackbar.setAction(R.string.scansimpleid_error_state_snackbar_ok) { invalidQrCodeSnackbar.dismiss() }.show()
```

Comment:
When the action button is clicked, it will automatically get dismissed, so we don't have to do that manually. You can just leave it as an empty lambda.